deploy/olm-catalog/ibm-block-csi-operator/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml

```diff
@@ -209,7 +209,7 @@ spec:
     #### 2.  Install the IBM block storage CSI driver by creating an IBMBlockCSI custom resource.
     ##### 2.1.  Download the manifest from GitHub.
     ```bash
-    curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.7.0/deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml > csi.ibm.com_v1_ibmblockcsi_cr.yaml
+    curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.7.0/config/samples/csi.ibm.com_v1_ibmblockcsi_cr.yaml > csi.ibm.com_v1_ibmblockcsi_cr.yaml
```

Comment:
do we want to update all the documentation urls in this PR?
and if not, do we have a ticket for it?

Reply:
this is considered documentation? because it's in the csv

Reply:
it's not code, and it's visible to the user in the GUI.
we usually update this with the documentation

Reply:
but we already changed it to 1.7.0 and in the readme files of the project we didn't so they don't change in the same time

Reply:
the CSVs were not changed to 1.7.0, they were created with it.
and I don't see e.g. `ibm-block-csi-operator-community` in this PR anyway, so I think it is best to reduce the diff here and open a separate ticket for readme, CSV, documentation, etc...